Rakefile

```diff
@@ -1,12 +1,12 @@
 namespace :build do
   desc 'Builds the Epoxy package'
   task :package do
-    sh 'xcodebuild build -scheme Epoxy -destination generic/platform=iOS'
+    sh 'xcodebuild build -scheme Epoxy -destination generic/platform=iOS | mint run xcbeautify'
```

Comment:
should we make a little function like this?
```ruby
def pretty_sh(cmd) do
  sh "#{cmd} | mint run xcbeautify"
end
```

Reply:
Good call, added in https://github.com/airbnb/epoxy-ios/pull/65/commits/3c63e2f4371618c24e51c7a756e86c69238a4658